src/test/suite/grammar.test.ts

```diff
@@ -37,8 +36,6 @@ const registry = new vsctm.Registry({
 });
 
 suite('Grammar Test Suite', () => {
-    vscode.window.showInformationMessage('Start all tests.');
```

Comment:
This is unrelated, right?

Reply:
Kinda unrelated: I spotted it when creating the logger file test.  This message is pretty much useless, probably taken from an example test.  I thought it wouldn't be _too_ bad to remove these lines here, this won't even hurt git-blaming :upside_down_face: